### PR TITLE
Update Simplified Chinese to v1.1

### LINKED ISSUE #1028: Site Translation: Update Simplified Chinese to v1.1
### Prize Bounty
40 DAI

### Challenge Description
Simplified Chinese v1.1 translations have completed review & are ready to be added to the website!

### Submission Requirements

1. Download the [files from CrowdIn](https://crowdin.com/project/ethereumfoundation/zh-CN#) for this version of the content (`Contents v.1.1`). You may need to sign up for CrowdIn & join our project in order to download the files - here's the [invite link](https://crowdin.com/project/ethereumfoundation/invite).
2. Replace all files within the /docs/translations/zh/ directory & add any new files, keeping the same directory structure as CrowdIn (i.e. each page is a subdirectory with an index.md filename, a strings.json & an index.md at the root of the directory)
3. In each newly added markdown file...
- Within the [front matter](https://vuepress.vuejs.org/guide/frontmatter.html):
  - Confirm each file has `description`, `og:title` & `og:description` meta properties - check out other translation markdown files for reference
  - Confirm each file has the correct `lang` property (it should match the `locales` key in config.js - `zh-CN`)
- Update all relative links to include the language path. e.g. a link to the /learn page within the /docs/translations/sv/what-is-ethereum/index.md file should link to ethereum.org/sv/learn, not ethereum.org/learn. You should be able to use our script to accomplish this (i.e. `yarn update-internal-links zh`).
4. Update the `version` key in the /utils/translations.js file for this translation (`zh-CN`) to `1.1`
5. Update the site navigation for the translation in config.js (`locales` key) to match the structure of other v1.1 translations (e.g. Swedish, `/se/`). For reference, you can see the site version of each language in translations.js `version` key.

**Additional context**

If you'd like additional guidance, see how #816 solved #778 for a similar translation

### Judging Criteria
The prize will be awarded to the first valid submission.

### PR DESCRIPTION
**Pull request by sebastiantf**
## Description
Updates Simplified Chinese to v1.1

## Related Issue
Fix #1028

### CODE REVIEW

File: docs/translations/zh/build/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (samajammin) [inline docs/translations/zh/build/index.md:1]: This page is incorrect. I just downloaded the [file from CrowdIn](https://crowdin.com/project/ethereumfoundation/zh-CN#) (Contents v1.1). This is the content of the file:
```
---
title: 开始搭建
meta:
  - 
    name: 说明
    content: 通过Ethereum Studio了解以太坊基础知识。以太坊Studio是基于网页的IDE，可以用来搭建及测试智能合约，并为其搭建一个前端
  - 
    property: og:标题
    content: 开始搭建 | 以太坊
  - 
    property: og:说明
    content: 通过 Ethereum Studio 了解以太坊基础知识。Ethereum Studio 是基于网页的 IDE，可以用来搭建并测试智能合约。
sidebar: false
layout: 首页
hideHero: true
lang: zh-CN
---

<BuildPage />
```
Please ensure all files have been correctly downloaded.

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -869,17 +869,69 @@ module.exports = {
     },
     '/zh/': {
       lang: 'zh-CN',
-      label: translate('language', 'zh-CN'),
-      title: translate('ethereum', 'zh-CN'),
+      label: translate('language'),
```
Comment (samajammin) [inline docs/.vuepress/config.js:872]: This should be `label: translate('language', 'zh-CN'),`

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -869,17 +869,69 @@ module.exports = {
     },
     '/zh/': {
       lang: 'zh-CN',
-      label: translate('language', 'zh-CN'),
-      title: translate('ethereum', 'zh-CN'),
```
Comment (samajammin) [inline docs/.vuepress/config.js:873]: We don't want to delete this `title` key.

File: docs/translations/zh/developers/index.md [hunk 4]
```diff
@@ -1,11 +1,15 @@
 ---
-title: 开发者
+title: Ethereum for Develop
```
Comment (samajammin) [inline docs/translations/zh/developers/index.md:2]: This file is also incorrect, this is the frontmatter content I see after downloading the file:
```
---
title: 面向开发者的以太坊资源
meta:
  - 
    name: 说明
    content: 本页为以太坊开发者提供入门指南、学习资源以及开发工具。
  - 
    property: og:标题
    content: 面向开发者的以太坊资源
  - 
    property: og:说明
    content: 本页为以太坊开发者提供入门指南、学习资源以及开发工具。
lang: zh-CN
sidebar: 自动
sidebarDepth: 1
---
```
Please ensure all files have been correctly downloaded.

File: docs/translations/zh/dot-net/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+---
```
Comment (samajammin) [inline docs/translations/zh/dot-net/index.md:1]: Just spot-checked this file - this is also incorrect. Please ensure you're downloading & using the Contents v1.1 files.

File: docs/translations/zh/strings.json [hunk 6]
```diff
@@ -1,33 +1,75 @@
...
   "page-home-section-use-item-one": "我能用以太坊做什么",
-  "page-home-section-use-item-three": "什么是钱包？",
+  "page-home-section-use-item-three": "What's a wallet?",
   "page-home-section-use-item-two": "如何获得 ETH",
   "page-home-section-use-title": "使用",
   "page-home-subtitle": "在以太坊上，你可以通过编写代码管理数字资产、运行程序，更重要的是，这一切都不受地域限制。",
-  "page-home-title": "以太坊是一个为去中心化应用（Dapp）而生的全球开源平台。",
-  "page-last-updated": "页面上次更新",
+  "page-home-title": "以太坊是一个为去中心化应用（DApp）而生的全球开源平台。",
+  "page-individuals": "Individuals",
```
Comment (samajammin) [inline docs/translations/zh/strings.json:68]: This is also the wrong/outdated file. Just checked & this key should be `"个人"`

File: docs/.vuepress/config.js (hunk 3 above)
Comment (sebastiantf) [reply docs/.vuepress/config.js:873]: I was using the /se/ in config.js as a reference and this is how it looks: https://github.com/ethereum/ethereum-org-website/blob/3ece6c6fb9fc63bd529dc39448735ba374d48cad/docs/.vuepress/config.js#L669

```
'/se/': {
      lang: 'sv-SE',
      label: translate('language'),
      nav: [
      ...
```
This is how it looks for other v1.1 translations too

File: docs/.vuepress/config.js (hunk 2 above)
Comment (sebastiantf) [reply docs/.vuepress/config.js:872]: Please see my comment above.

File: docs/.vuepress/config.js (hunk 3 above)
Comment (sebastiantf) [reply docs/.vuepress/config.js:873]: Also in #816 I see the same way

**Comment by sebastiantf** [comment]
@samajammin I have made the changes requested. Please review. Also please see my comments on your requests

**Comment by sebastiantf** [comment]
@samajammin It seems there are differences in the files you get by downloading a single file from CrowdIn (by specifically opening a file and downloading it) and the files you get by downloading the whole folder by clicking that download button on the top right. Is that normal? So I should open and download each file? I was downloading the whole folder and it worked fine when doing the last few updations.

File: docs/.vuepress/config.js (hunk 3 above)
Comment (samajammin) [reply docs/.vuepress/config.js]: Thanks! You're right - I don't believe this is actually needed.

Comment (sebastiantf) [reply docs/.vuepress/config.js]: Should I revert the commit I did?

**Comment by samajammin** [comment]
> @samajammin It seems there are differences in the files you get by downloading a single file from CrowdIn (by specifically opening a file and downloading it) and the files you get by downloading the whole folder by clicking that download button on the top right. Is that normal? So I should open and download each file? I was downloading the whole folder and it worked fine when doing the last few updations.

@sebastiantf thanks for bringing that up! I just tried & had the same issue. I've contact CrowdIn support about this. I'll let you know if they have a solution.

**Comment by sebastiantf** [comment]
@samajammin I already did go through each file and replaced it in my latest commits.

Also, should I revert https://github.com/ethereum/ethereum-org-website/pull/1029/commits/36b32a4030ff265862cd59af9bc90db006cdf838

**Comment by samajammin** [comment]
@sebastiantf no worries, I'll remove that in a separate PR. Thanks for the contribution! Glad to see you back 😄

**Comment by sebastiantf** [comment]
Cool. Thanks for the merge. Happy to be back too. Looking forward for more opportunities 😄

**Comment by samajammin** [comment]
FYI @sebastiantf I figured out the issue with the outdated CrowdIn files. #1039 should fix this problem.

Thanks again for pointing it out!

**Comment by sebastiantf** [comment]
Glad u could triage it. 👍🏻

**Pull request merged**